lms/static/js/student_account/views/account_settings_view.js

```diff
@@ -1,29 +1,67 @@
 ;(function (define, undefined) {
     'use strict';
     define([
-        'gettext', 'jquery', 'underscore', 'backbone', 'text!templates/student_account/account_settings.underscore'
-    ], function (gettext, $, _, Backbone, accountSettingsTemplate) {
+        'gettext',
+        'jquery',
+        'underscore',
+        'backbone',
+        'js/student_account/views/account_section_view',
+        'text!templates/student_account/account_settings.underscore'
+    ], function (gettext, $, _, Backbone, AccountSectionView, accountSettingsTemplate) {
 
         var AccountSettingsView = Backbone.View.extend({
 
+            navLink: '.account-nav-link',
+            activeTab: 'aboutTabSections',
+            accountSettingsTabs: [
+                {name: 'aboutTabSections', id: 'about-tab', label: gettext('About'), class: 'active'},
```

Comment:
Is "About" really the best label for this tab?  When taken out of context, its unclear what About is About.  There is already an about link in the footer of the page (About edX).  So you can see how, if experienced out of context, this could be confusing.  "User Settings"  "Personal Settings" something like that?

cc @srpearce